pkg/ui/src/views/databases/containers/databases/nonTableSummary.tsx

```diff
@@ -38,7 +42,7 @@ class NonTableSummary extends React.Component<TimeSeriesSummaryProps> {
     this.props.refreshNonTableStats();
   }
 
-  renderTable() {
+  renderTable = () => {
```

Comment:
It is necessary to preserve the reference to `this` when passing function in props to `Loading` component.